docs/testing.md

```diff
@@ -132,13 +132,11 @@ If you want to work on a specific issue
 8.  make a PR to ocis running the adjusted tests
 
     To confirm that all tests (old and changed) run fine make a PR to ocis with your code changes and point drone to your branch in core to get the changed tests.
-    For that change this line in the `acceptance-tests` section
+    For that change this line in the `main` function
 
-    `'git clone -b master --depth=1 https://github.com/owncloud/core.git /srv/app/testrunner',`
+        apiTests(ctx, 'master', 'a3cac3dad60348fc962d1d8743b202bc5f79596b')
 
-    to clone your core branch e.g.
-
-    `'git clone -b fixRevaIssue122 --depth=1 https://github.com/owncloud/core.git /srv/app/testrunner',`
+    In place of master use your branch name. Also make sure to change the commit ID to the hash of the latest commit in your branch or any other commits you want to test.
```

Comment:
Or you can delete the commit id:
`apiTests(ctx, 'my-branch', '')`

and the CI will use the current head of the branch in core.